recipes/scine-molassembler/build.sh

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
+
+set -ex
+
+# No CMake find modules for dependencies provided, so we need to add them manually.
+cp -v $RECIPE_DIR/cmake/* cmake/
+
+# Fix fenv.h for OSX
+cp -v $RECIPE_DIR/include/* src/
+
+cmake \
+    $CMAKE_ARGS \
+    -B _build -G Ninja \
+    -DBUILD_SHARED_LIBS=ON \
+    -DSCINE_MARCH="" \
+    -DSCINE_USE_INTEL_MKL=OFF \
```

Comment:
Fine for now - you're the expert here - but if you want to speed this up, it would probably be good to add MKL or similar later on in the feedstock.

Reply:
As far as I can see this only affects the LAPACK/BLAS finder. We should get most of the speed-up from MKL by using `libblas=*=*mkl*` at runtime, since there is no dedicated code leveraging MKL specific routines if MKL is detected at build time.